kolibri/plugins/management/urls.py

```diff
@@ -4,4 +4,5 @@
 
 urlpatterns = [
     url('^$', views.ManagementView.as_view(), name='management'),
```

Comment:
Might be good to edit the facility url to `'^facility$'` just to make sure there are no collisions.